src/popup/views/not-authed.tsx

```diff
@@ -1,5 +1,16 @@
 import * as React from 'react'
 
 export function NotAuthed({ signInWithTwitter }: Pick<Dispatchers<UserAction>, 'signInWithTwitter'>): JSX.Element {
-  return <button onClick={signInWithTwitter}>Sign in with twitter</button>
+  return (
+    <div className="not-authed">
+      <img src="../../../img/roar_128.png" />
```

Comment:
`<img src="/img/roar_128.png" />` is enough. When we start the `src` with a "/" it looks in the project's root directory.

Reply:
updated both img elements path.